src/node-api-jsi/NodeApiJsiRuntime.cpp

```diff
@@ -1528,15 +1528,13 @@ jsi::Value NodeApiJsiRuntime::callAsConstructor(const jsi::Function &func, const
 }
 
 jsi::Runtime::ScopeState *NodeApiJsiRuntime::pushScope() {
-  NodeApiEnvScope scope{getEnv()};
```

Comment:
Do we pass all unit tests after this change?

Reply:
Yes. Seems like this was mismatched. enter1 -> enter2 -> exit1 -> exit2, and was crashing in V8 code.